db/migrate/20170203141631_add_maintenance_to_storages.rb

```diff
@@ -0,0 +1,5 @@
+class AddMaintenanceToStorages < ActiveRecord::Migration[5.0]
+  def change
+    add_column :storages, :maintenance, :boolean
```

Comment:
@gmcculloug no this is actually a property of the datastore not the mount, so a datastore cannot be in maintenance mode on some hosts but not others